cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -266,6 +268,8 @@
       /** for the address (hostname) of the dataset server. */
       public static final String ADDRESS = "dataset.service.bind.address";
       public static final String PORT = "dataset.service.bind.port";
+      public static final String ANNOUNCE_ADDRESS = "dataset.service.announce.address";
```

Comment:
okay, i think its fair to make that change in this PR. as it wouldn't make sense to have two separate config name for bind address and one config for announce address.